docs/exchanges.md

```diff
@@ -74,23 +74,13 @@ Should you experience constant errors with Nonce (like `InvalidNonce`), it is be
 $ pip3 install web3
 ```
 
-### Send incomplete candles to the strategy
+### Getting latest price / Incomplete candles
 
 Most exchanges return current incomplete candle via their OHLCV/klines API interface.
 By default, Freqtrade assumes that incomplete candle is fetched from the exchange and removes the last candle assuming it's the incomplete candle.
 
 Whether your exchange returns incomplete candles or not can be checked using [the helper script](developer.md#Incomplete-candles) from the Contributor documentation.
```

Comment:
```suggestion
Whether your exchange returns incomplete candles or not - it can be checked using [the helper script](developer.md#Incomplete-candles) from the Contributor documentation.
```

Reply:
really? to me this sounds strange with the seperating word (when trying to read it out loud...)

Reply:
> really? to me this sounds strange with the seperating word (when trying to read it out loud...)

no problem, that was only my suggestion...